sharedgo/deletionPicker.go

```diff
@@ -0,0 +1,21 @@
+// Picks what file from 10 to 50 (files 0-9 kept in the system for persistence) to delete
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func pick_file_to_delete() string {
+	rand.Seed(time.Now().UnixNano())
+	var file_ID int = rand.Intn(40) + 10;
```

Comment:
- You may want to use constants for 40 and/or 10 (e.g. MAX_DELETION_INDEX and DELETION_RATE), this should make it easier to update/modify if for some reason it's not feasible in the future.

Also make it sure to leave comments next to each constant declaration.
```suggestion
	file_ID := rand.Intn(40) + 10
```